package.json

```diff
@@ -36,14 +37,12 @@
   "devDependencies": {
     "@babel/eslint-parser": "7.17.0",
     "@babel/preset-react": "^7.16.7",
-    "@swc/core": "1.2.138",
-    "@swc/helpers": "0.3.2",
```

Comment:
The new approach to `yarn exe` uses `--experimental-loader ts-node/esm/transpile-only` instead of `ts-node` binary. We no longer need `@swc/*` which lightens our `node_modules` a bit. Transpiling is still fast enough for our purpose.